### DISCUSSION TITLE
maximum speed and acceleration

### DISCUSSION THREAD

**Discussion by familynygood** [pybricks/support#1813] [General]
What is the maximum speed, acceleration, turning speed and turning acceleration pybricks allows?

**Comment by FLL-Team-24277** [comment]
I saw this hadn't been answered, but I will do my best. I assume you are talking about a pybricks drivebase, but you didn't mention it. The problem with answering your question though, is you didn't tell us what size tires you are using or what motors. All of those actions are dependent on how you built your robot.

**Comment by DylanBPY** [comment]
All depends on how fast the motors can move, this is dependent on weight, surface, and countless other things.
